gtests/test_nopython_callprofiler.cpp

```diff
@@ -212,5 +212,124 @@ TEST_F(CallProfilerTest, cancel)
     EXPECT_EQ(radix_tree().get_unique_node(), 0);
 }
 
+std::string line2 = "";
```

Comment:
This kind of tests, which assert for printed texts, should be in Python, not in C++.